templates/title/show.json.twig

```diff
@@ -0,0 +1,46 @@
+{
+    "@context": "http://schema.org/",
+    "@type": "Book",
+    "@id": "{{ url('title_show', {'id':title.id}) }}",
+    "name": "{{title.title}}"
+    {% if title.pubdate %}, "datePublished": "{{title.pubdate}}" {% endif %},
+    "locationCreated": {
+        "@type": "Place"
+        {% if title.locationOfPrinting %}, "name": "{{title.locationOfPrinting}}" {% endif %},
+        "geo": {
+            "@type": "GeoCoordinates",
+            "latitude": "{{title.locationOfPrinting.latitude}}",
+            "longitude": "{{title.locationOfPrinting.longitude}}"
+        }
+    }
+    {% if title.locationOfPrinting %}, "publisherImprint": "{{title.imprint}}" {% endif %},
```

Comment:
Two things:

1. You have locationOfPrinting here--is that meant to be imprint?
1. I'm not sure publisherImprint is what we want here. Though the name sounds like it's exactly what we need, the description from the documentation seems to say otherwise (https://schema.org/publisherImprint):

> The publishing division which published the comic.

It expects an a child property, so from what I can gather, it's really meant to describe a specific imprint of a publisher (i.e. a subdivision or a particular subsection of a larger publishing firm). In this project, an imprint is a longer text description that is printed on the title page of pre-18th century books (so something like "Printed by John Smith at the Sign of the Boar on Fleet Street"). I'm not sure if there's a good way to capture that in the schema.org schema.

I would suggest just dropping this property entirely unless we can find something that suits our purposes a bit better.

Reply:
1)Yes it was meant to be imprint! I just copied the if clause from the above statement :D 
2)Ok I will drop that :)